front/src/modules/pathfinding/utils.ts

```diff
@@ -79,6 +79,7 @@ export const getPathfindingQuery = ({
         rolling_stock_supported_signaling_systems: rollingStock.supported_signaling_systems,
         rolling_stock_maximum_speed: rollingStock.max_speed,
         rolling_stock_length: rollingStock.length,
+        stop_at_next_signal: true, // TODO: change to false and set to true elsewhere if needed
```

Comment:
```suggestion
        stop_at_next_signal: false,
```

We keep this flag to `true` for review purpose, but before merging, we need to set it to `false`